src/libcrun/cgroup-resources.c

```diff
@@ -937,7 +938,17 @@ write_cpu_resources (int dirfd_cpu, bool cgroup2, runtime_spec_schema_config_lin
           len = sprintf (fmt_buf, "%" PRIu64, cpu->period);
           ret = write_cgroup_file (dirfd_cpu, "cpu.cfs_period_us", fmt_buf, len, err);
           if (UNLIKELY (ret < 0))
-            return ret;
+            {
+              if (errno != EINVAL)
```

Comment:
could you please replace `errno` with `crun_error_get_errno (err)`?

Reply:
That's exactly what I was thinking, thanks!

Reply:
Right (I thought about that but forgot).

In addition, I forgot to add `!cpu->quota` check, since if we won't be setting quota later, we need to return an error. Fixed now.